perl-libs/packaging/centreon-perl-libs-common.yaml

```diff
@@ -54,6 +54,12 @@ overrides:
       - libdigest-sha-perl
       - libgd-perl
       - libcrypt-urandom-perl
+    replaces:
+      - centreon-perl-libs (<< 25.10.0)
```

Comment:
I think in the past, in the rpm world, we (rpm packages, not Centreon) were using Provides + Obsoletes to do this. Not Replaces. I'd need to double-check, though...

(not clear about debian)